app/_includes/gateway-support-third-party.html

```diff
@@ -0,0 +1,120 @@
+
+## Data Stores
+
+Kong interacts with multiple data stores during normal operation. The following stores have been tested by Kong:
+
+
```

Comment:
The following blocks are interactive (mouse over highlights block), which usually implies interactivity (clickable / navigable).  I think it is slightly confusing to have this highlight function.